.github/pull_request_template.md

```diff
@@ -18,11 +19,12 @@ not fully covered by unit tests or manual testing can be complicated. -->
 
 ### Checklist
 <!-- Put an 'x' in all the boxes that apply -->
-- [ ] I submit my changes into the `develop` branch
-- [ ] I have added description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md)
+- [ ] I have added unit tests to cover my changes.​
+- [ ] I have added integration tests to cover my changes.​
+- [ ] I have added e2e tests for validation.
+- [ ] I have added the description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/releases/1.1.0/CHANGELOG.md).​
```

Comment:
should we update this for every release?

Reply:
> should we update this for every release?
I think so. If RC branch is created, if there are some changes, the default PR 
 template will link to develop's one. I think we should link the proper version's one.  @yunchu What do you think of it? If not, every default template will link to develop branch.